site/src/components/Markdown/Markdown.tsx

```diff
@@ -91,12 +97,50 @@ export const Markdown: FC<{ children: string }> = ({ children }) => {
   )
 }
 
+export const MemoizedMarkdown = React.memo(Markdown)
```

Comment:
Created a memoized version so this is not going to be reloaded all the time since we have auto-update in the template page. I also memorized this to make my life easier on debug - the component was refreshing and losing the styles that applied in the browser every 2 seconds 😅